modules/persistent-storage-csi-vsphere-encryption-tag-based.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
```

Comment:
Same note as in the other module - please use the complete path to the assembly.